### DISCUSSION TITLE
How to Switch Between Multiple Firestore Databases in the same firebase app.

### DISCUSSION THREAD

**Discussion by Harikarthyk** [invertase/react-native-firebase#7943] [Q&A]
My Firebase project contains multiple Firestore databases, and I need to switch between these databases in my code.
 
```
 const response = await firestore().collection(collectionId).doc(savedId).get().then((doc: any) => {
      console.log(doc.data(), 'doc.data()');
      return doc.data();
});
```

This connects to the default database, but how to mention the database Id ?

**Comment by anurag-lio** [comment]
@Harikarthyk were you able to figure this out?

**Reply by Harikarthyk** [reply]
@anurag-lio  No, I could not find any solution. I have also reached out to the Firestore to make my current Firestore database a default database, but there are no such options available currently, so I have migrated the entire database documents to the default database. In my case, I am not going to query in the default database, so I swapped both databases, but in case you need to query both databases, then there is no such solution as of now, I guess.
